content/applications/productivity/studio/pdf_reports.rst

```diff
@@ -0,0 +1,340 @@
+===========
+PDF reports
+===========
+
+Odoo uses HTML and CSS technologies to create reports (PDF documents). HTML elements provide the
+basic structure, while CSS interacts with the elements to establish how they are shown to the user.
+Odoo also uses Bootstrap, a CSS framework, for the grid layout.
+
+.. _studio/pdf-reports/default-layout:
+
+Default layout
+==============
+
+The default layout of reports is managed outside Studio. Go to :menuselection:`Settings -->
+Companies: Document Layout --> Configure Document Layout`. Layout settings apply to all reports but
+only to the current company.
+
+.. tip::
+   Use :guilabel:`Download PDF Preview` to view how the different settings affect the layout of a
+   sample invoice.
+
+.. _studio/pdf-reports/default-layout/layout:
+
+Layout
+------
+
+Four layouts are available.
```

Comment:
is that a useful inclusion? since it's not configured in studio anyway?

Reply:
(less content now = less screenshots to update in the future :grimacing: )

Reply:
I like the different previews so I think I will keep them😅